cinema/views.py

```diff
@@ -19,27 +21,57 @@
     OrderSerializer,
     OrderListSerializer,
 )
+from cinema.permissions import IsAdminOrIfAuthenticatedReadOnly
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class GenreViewSet(
+    viewsets.GenericViewSet,
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
 
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
 
-class ActorViewSet(viewsets.ModelViewSet):
+
+class ActorViewSet(
+    viewsets.GenericViewSet,
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+):
     queryset = Actor.objects.all()
     serializer_class = ActorSerializer
 
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
```

Comment:
Set `TokenAuthentication` and `IsAdminOrIfAuthenticatedReadOnly` as defaults
https://www.django-rest-framework.org/api-guide/authentication/#setting-the-authentication-scheme
https://www.django-rest-framework.org/api-guide/permissions/#setting-the-permission-policy